source/Integrate/Code_Examples/index.html

```diff
@@ -7,54 +7,42 @@
 ---
 
 <div class="row">
-  <ul>
-    <li>
-      <a href="csharp.html">
-        C# 
-        <span class="icon-net" alt="C#"></span>
-      </a>
-    </li>
-    <li>
-      <a href="go.html">
-          Go 
-        </a>
-    </li>
-    <li>
-      <a href="java.html">
-        Java
-        <span class="icon-java"></span>
-      </a>
-    </li>
-    <li>
-    	<a href="nodejs.html">
-        Node.js
-        <span class="icon-nodejs"></span>
-      </a>
-    </li>
-    <li>
-      <a href="perl.html">
-        Perl
-        <span class="icon-perl"></span>
-      </a>
-    </li>
-    <li>
-      <a href="php.html">
-        PHP
-        <span class="icon-php"></span>
-      </a>
-    </li>
-    <li>
-      <a href="python.html">
-        Python
-        <span class="icon-python"></span>
-      </a>
-    </li>
-    <li>
-      <a href="ruby.html">
-        Ruby
-        <span class="icon-ruby"></span>
-      </a>
-    </li>
-  </ul>
+  <div class="col-md-8">
+      <h2><a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/">Web API v2 Code Examples</a></h2>
+      <p>
+      Learn how to quickly start sending email through SendGrid's Web API v2. Here you will find code examples for our v2 Mail Send endpoint in 8 different languages.
+      </p>
+  </div>
+  <div class="col-md-4 subsection-links">
+    <br />
+    <p class="subsection-title">Libraries</p>
+      <p><a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/csharp.html">C#<span class="icon-net" alt="C#"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/go.html">Go</a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/java.html">Java<span class="icon-java"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/nodejs.html">Node.js<span class="icon-nodejs"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/perl.html">Perl<span class="icon-perl"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/php.html">PHP<span class="icon-php"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/python.html">Pyton<span class="icon-python"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/ruby.html">Ruby<span class="icon-ruby"></span></a></p>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-md-8">
+      <h2><a href="{{root_url}}/Integrate/Code_Examples/v2_Mail/">Web API v3 Code Examples</a></h2>
+      <p>
+      Learn how to quickly start sending email through SendGrid's Web API v3. Here you will find code examples for our latest v3 Mail Send endpoint in 7 different languages.
+      </p>
+  </div>
+  <div class="col-md-4 subsection-links">
+    <br />
+    <p class="subsection-title">Libraries</p>
+      <p><a href="{{root_url}}/Integrate/Code_Examples/v3_Mail/csharp.html">C#<span class="icon-net" alt="C#"></span></a><br />
+      <a href="{{root_url}}/Integrate/Code_Examples/v3_Mail/go.html">Go<br />
```

Comment:
Would be cool to get a Go icon: https://www.google.com/search?q=golang+icon&biw=1882&bih=1323&tbm=isch&tbo=u&source=univ&sa=X&ved=0ahUKEwj8_tbIot_MAhUO3mMKHaeTD1sQsAQIGw